translations/en-us.yml

```
@@ -1352,6 +1352,8 @@ registries:
         submit: 'Submit'
         success: 'Successfully submitted moderator decision.'
         failure: 'Failed to submit moderator decision.'
+        justificationForWithdrawal: 'Justification for Withdrawal'
+        remarksToAdmin: 'Remarks to registration admin'
```

nit-pick. feel free to ignore this
```suggestion
        remarksToAdmin: 'Add remarks to the registration admins'
```